private/config-variables.js

```diff
@@ -27,7 +25,8 @@ function createConfig(name, slug = null, config = {}) {
     },
 
     plugins: [
-      new webpack.BannerPlugin(`MyParcel ${name} ${process.env.VUE_APP_VERSION}`),
+      // eslint-disable-next-line vue/max-len,max-len
+      new webpack.BannerPlugin(`MyParcel ${name} ${process.env.npm_package_version} [${process.env.npm_package_gitHead}]`),
```

Comment:
Ik ga het toch zeggen: dit hoort in een andere story/branch.

Reply:
ugh 😢